aio/content/marketing/contributors.json

```
@@ -856,5 +856,13 @@
     "bio": "Doug is an overly-opinionated software developer with a passion for making awesome developer tools. He is motivated primarily by his selfish desire to make his own life easier, but loves to share his tools and workflows with others to make everyone's lives easier.",
     "groups": ["Angular"],
     "lead": "igorminar"
+  },
+  "santosh": {
+    "name": "Santosh Yadav",
+    "picture": "santoshyadav.jpg",
+    "twitter": "Santosh19742211",
+    "website": "https://www.santoshyadav.dev",
+    "bio": "Santosh is GDE for Angular and Web Technologies and loves to contribute to Open Source. He is the creator of ng deploy for netlify and core team member for NestJS Addons. He writes for AngularInDepth and mentors for DotNetTricks, he is the organizer for Pune Tech Meetup and conducts the free workshops on Angular.",
```

```suggestion
    "bio": "Santosh is a GDE for Angular and Web Technologies and loves to contribute to Open Source. He is the creator of ng deploy for netlify and core team member for NestJS Addons. He writes for AngularInDepth, mentors for DotNetTricks, organizes the Pune Tech Meetup, and conducts free workshops on Angular.",
```

For "organizes the Pune Tech Meetup", "the" is optional if its official name is "Pune Tech Meetup", but if it's called "Tech Meetup" that happens to be located in Pune, then you'd definitely want "the". Having "the" can help with clarity in either case.

---

Thanks @kapunahelewong and done with changes.